### PR TITLE
feat: update deps, local-only skate

### PR DESCRIPTION
**Pull request by caarlos0**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -88,32 +90,17 @@ var (
...
-		RunE:  sync,
-	}
-
-	resetCmd = &cobra.Command{
-		Use:   "reset [@DB]",
-		Short: "Delete local db and pull down fresh copy from Charm Cloud.",
-		Args:  cobra.MaximumNArgs(1),
-		RunE:  reset,
-	}
-
```
Comment (aymanbagabas) [inline main.go:104]: I think we should leave this and add a warning in the long description that the service will go down soon. This give people some time to pull their data using `sync`

**Comment by meowgorithm** [comment]
Do we still need the Dockerfile?

File: main.go (hunk 1 above)
Comment (caarlos0) [reply main.go:104]: they can use the previous version for that... same thing for glow...

**Comment by caarlos0** [comment]
migration script, assuming the previous version of skate available as `./old-skate`:

```bash
#!/bin/bash
./old-skate list-dbs | while read -r db; do
  ./old-skate list -k "$db" | while read -r key; do
    ./old-skate get "$key$db" | skate set "$key$db"
  done
done
```

**Pull request merged**